### PR TITLE
Fix: Docblocks

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] fixes docblocks

Follows #744.
Follows #745.

### CODE REVIEW

File: classes/Infrastructure/Persistence/IlluminateSpeakerRepository.php [hunk 1]
```diff
@@ -19,15 +19,6 @@ public function __construct(User $userModel)
...
 
-    /**
-     * Retrieves a speaker with associated talks.
-     *
-     * @param string $speakerId
-     *
-     * @throws EntityNotFoundException
-     *
-     * @return User the speaker that matches given identifier
-     */
```
Comment (localheinz) [inline classes/Infrastructure/Persistence/IlluminateSpeakerRepository.php]: No need to repeat the docblock from the interface.

File: classes/Infrastructure/Persistence/IlluminateSpeakerRepository.php [hunk 2]
```diff
@@ -39,11 +30,6 @@ public function findById($speakerId): User
         return $speaker;
     }
 
-    /**
-     * Saves a speaker and their talks.
-     *
-     * @param User $speaker
-     */
```
Comment (localheinz) [inline classes/Infrastructure/Persistence/IlluminateSpeakerRepository.php]: No need to repeat the docblock from the interface.

File: classes/Infrastructure/Persistence/IlluminateTalkRepository.php [hunk 3]
```diff
@@ -7,9 +7,6 @@
 
 class IlluminateTalkRepository implements TalkRepository
 {
-    /**
-     * @param Talk $talk
```
Comment (localheinz) [inline classes/Infrastructure/Persistence/IlluminateTalkRepository.php]: Docblock doesn't add any value here, just repeats the type declaration.

**Comment by chartjes** [comment]
Thanks @localheinz

**Comment by localheinz** [comment]
Thank you, @chartjes!

**Pull request merged**